### PR TITLE
Alert Component Reviews.

### PR DESCRIPTION
**Pull request by m26unkwn**
Hey Neog Programmers

Need some reviews in my alert component.

Please don't hesitate to write your reviews.

### CODE REVIEW

**Comment by rahulyadav139** [comment]
Clean and minimilistic UI. Components are pleasing to the eyes. Great work.

File: styles/alert.css [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  color: var(--info-color);
+}
+
+.outline-success {
+  background-color: #fff;
+  border: 1px solid var(--success-color);
+  color: var(--success-color);
+}
+.outline-warning {
+  background-color: #fff;
```
Comment (rahulyadav139) [inline styles/alert.css]: instead of repeating color each time, you could this as a variable.

Comment (m26unkwn) [reply styles/alert.css]: thanks, Rahul will definitely add this.

Comment (m26unkwn) [reply styles/alert.css]: Just added the white-color varible

**Review by PuneetP16 — APPROVED** [review]
Looks good.

**Pull request merged**